### PR TITLE
Removes the MANIFEST file and unused release files

### PR DESCRIPTION
**Pull request by derobins**

### CODE REVIEW

File: bin/bbrelease [hunk 1]
```diff
@@ -330,64 +278,62 @@ for f in README.md release_docs/RELEASE.txt; do
...
     DOC_URL=https://git@bitbucket.hdfgroup.org/scm/hdffv/hdf5doc.git -b ${DOCVERSION}
 fi
 
 # Create the tar file
 test "$verbose" && echo "   Running tar..." 1>&2
 ( \
-    cd $tmpdir; \
-    tar cf $HDF5_VERS.tar $HDF5_IN_VERS/Makefile \
-	`sed 's/^\.\//'$HDF5_IN_VERS'\//' $MANIFEST` || exit 1 \
+    tar --transform "s,^\.,$HDF5_IN_VERS," -c --exclude=".git" -f "$tmpdir/$HDF5_VERS.tar" Makefile . || exit 1 \
```
Comment (derobins) [inline bin/bbrelease]: I believe --transform is a gnu extension to tar. Not sure how that will fare on the BSDs or MacOS.

Comment (qkoziol) [reply bin/bbrelease]: "man tar" on MacOS doesn't list a --transform option.

**Comment by qkoziol** [comment]
I think the 'tar' command needs to be addressed, but otherwise is a good cleanup.

**Review by qkoziol — APPROVED** [review]

**Comment by derobins** [comment]
> I think the 'tar' command needs to be addressed, but otherwise is a good cleanup.

I'll try it on MacOS over the weekend and I'll look at the man pages for the BSDs.

**Review by soumagne — APPROVED** [review]

**Review by byrnHDF — APPROVED** [review]

**Review by jhendersonHDF — COMMENTED** [review]
Approved, outside of also agreeing that we need to find a solution for the tar issue here

**Review by qkoziol — APPROVED** [review]

File: bin/bbrelease (hunk 1 above)
Comment (derobins) [reply bin/bbrelease]: Fixed. Going through the symlink using -h should be portable.

**Review by lrknox — APPROVED** [review]

**Pull request merged**